django-stubs/conf/global_settings.pyi

```diff
@@ -61,6 +61,8 @@ LANGUAGE_COOKIE_NAME: str = ...
 LANGUAGE_COOKIE_AGE: Optional[int] = ...
 LANGUAGE_COOKIE_DOMAIN: Optional[str] = ...
 LANGUAGE_COOKIE_PATH: str = ...
+LANGUAGE_COOKIE_HTTPONLY: bool = ...
+LANGUAGE_COOKIE_SAMESITE: Literal["Lax", "Strict", "None", False] = ...
```

Comment:
This is definitely wrong—the default value *is* `None`, so you *can* read `settings.LANGUAGE_COOKIE_SAMESITE` and get `None`, so this annotation breaks type safety.

Reply:
- #1246